recipes/ditaa/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# if some command fails, abort immediately
+set -ex
+
+mkdir -vp ${PREFIX}/bin;
+mkdir -vp ${PREFIX}/share/${PKG_NAME}-${PKG_VERSION};
```

Comment:
Why are you including the version in the folder?

Reply:
Updated. Version is removed from the name